mllib/src/main/scala/org/apache/spark/ml/r/LogisticRegressionWrapper.scala

```diff
@@ -97,7 +97,15 @@ private[r] object LogisticRegressionWrapper
       standardization: Boolean,
       thresholds: Array[Double],
       weightCol: String,
-      aggregationDepth: Int
+      aggregationDepth: Int,
+      lrow: Int,
+      lcol: Int,
+      urow: Int,
+      ucol: Int,
```

Comment:
See my comment above, we can merge the four parameters into two: ```numRowsOfBoundsOnCoefficients``` and ```numColsOfBoundsOnCoefficients```, please follow Scala naming convention.